### PR TITLE
Add badge in navbar for when user has directoryCare items

### PR DESCRIPTION
**Pull request by oBlissing**
* Add badge
* Added optional 'html' option to tab-menu tablist items
* Added some badge-color classes
* Added `brand-accent` less variable ( #D56381 )
* Added color to active flag tag

screen shot 2019-03-07 at 15 53 59

### CODE REVIEW

File: viewer/vue-client/src/components/layout/navbar.vue [hunk 1]
```diff
@@ -7,23 +9,27 @@ export default {
...
-    },
-    settings() {
-      return this.$store.getters.settings;
+    ...mapGetters([
+      'userCare',
+      'settings',
+      'user',
+    ]),
+    tabs() {
+      const $directoryCareBadge = this.userCare.length === 0 ? '' : `<span class="badge badge-accent">${this.userCare.length}</badge>`;
```
Comment (jesperengstrom) [inline viewer/vue-client/src/components/layout/navbar.vue]: How about lifting badge up a bit?

```suggestion
      const $directoryCareBadge = this.userCare.length === 0 ? '' : `<span class="badge badge-accent" style="position:relative; top:-6px;">${this.userCare.length}</badge>`;
```

Comment (oBlissing) [reply viewer/vue-client/src/components/layout/navbar.vue]: Haven't tried. Can you post a screenshot?

Comment (jesperengstrom) [reply viewer/vue-client/src/components/layout/navbar.vue]: after
:)

Comment (oBlissing) [reply viewer/vue-client/src/components/layout/navbar.vue]: IMO the badge should align with the rest of the line.

https://getbootstrap.com/docs/3.3/components/#Badges

If we want it to have more of a "bubble" look, we can align it up a bit but then we should also decrease the size.

Comment (jesperengstrom) [reply viewer/vue-client/src/components/layout/navbar.vue]: Ok fair enough.

**Pull request merged**